### DISCUSSION TITLE
Adding fields for sign up

### DISCUSSION THREAD

**Discussion by Graditude-Dev** [janko/rodauth-rails#130] [Q&A]
What is the best way to add additional fields for signing up, i.e. username, first name, etc?

**Comment by janko** [comment]
[Here is an example](http://rodauth.jeremyevans.net/rdoc/files/doc/guides/registration_field_rdoc.html) from official guides. In rodauth-rails you would generate view templates for signup (`rails g rodauth:views create_account`), and add the field to the form. Both the [official demo](https://github.com/janko/rodauth-demo-rails) and the [intro screencast](https://youtu.be/2hDpNikacf0?t=310) show an example.
